packages/cdk-assets/test/docker-images.test.ts

```diff
@@ -183,7 +183,7 @@ describe('external assets', () => {
 
     const expectAllSpawns = mockSpawn(
```

Comment:
We need an assertion that verifies that the default is to run next to `assets.json`, right?

Reply:
Yes, that's why I extended the mock of "sometool" so that it also checks that cwd is set to "/external/cdk.out".